website/docs/r/opc_compute_storage_volume.html.markdown

```diff
@@ -78,6 +78,42 @@ Storage Volume's can be imported using the `resource name`, e.g.
 $ terraform import opc_compute_storage_volume.volume1 example
 ```
 
+## Snapshots
+
+Restoring a storage volume from a snapshot can happen via the use of the `snapshot`, `snapshot_id`, and `snapshot_account` arguments.
+
+Note: There isn't a mechanism to restore a snapshot from a different domain, the `snapshot_account` argument needs to reference the same domain.
```

Comment:
can we change `Note: ` for `~> **Note**: ` to make this a yellow box? Also perhaps `At this time there isn't a mechanism ...`?